spec/classroom_spec.rb

```diff
@@ -0,0 +1,13 @@
+require 'spec_helper'
+
+describe Classroom do
+  before :each do
+    @classroom = Classroom.new('Label')
+  end
+
```

Comment:
- [ ] It would be better if we test the `add_student` in our classroom class. Currently we have no tests for that :+1: 
![Screenshot from 2022-09-14 14-03-17](https://user-images.githubusercontent.com/46388113/190175767-2cf693cf-82cd-4f87-8d9c-eccc25042110.png)
For more clarity.....

Reply:
This test has been created